sample/src/main/res/drawable/ic_language_black_24px.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
JFYI: You might have known.
If you use [this chrome plugin](https://github.com/jmatsu/vector-drawable-previewer), you can see VectorDrawable icon on GitHub page like this. It helps PR review a lot.

<img width="400" src="https://user-images.githubusercontent.com/1269214/26856364-2c567c3c-4b5e-11e7-9bc1-f96711bdb312.png">